source/js/media/types/TL.Media.Twitter.js

```diff
@@ -61,14 +100,36 @@ TL.Media.Twitter = TL.Media.extend({
 		tweet_status_date 	= tweet_status_temp.split("\"\>")[1].split("<\/a>")[0];
 		
 		// Open links in new window
-		tweet_text = tweet_text.replace(/<a href/ig, '<a class="tl-makelink" target="_blank" href');
+		tweet_text = tweet_text.replace(/<a href/ig, '<a target="_blank" href');
+        
+        if (tweet_text.includes("pic.twitter.com")) {
+            
+            twttr.ready(
+                function(evt) {
```

Comment:
This anonymous function doesn't seem to be closed out properly. I am getting a build error caused by the unexpected else on line 124 due to missing close on this function